style.module.scss

```diff
@@ -1,3 +1,82 @@
+.overviewInstallChoice, .overviewFeatureChoice {
+  margin-top: 15px;
+}
+.overviewInstallChoice + h2, .overviewFeatureChoice + h2 {
+  margin-top: 32px;
+}
+.overviewInstallChoiceStep, .overviewFeatureChoiceStep {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: left;
```

Comment:
This is [not a valid value for this property](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#syntax). I'd suggest an alternative, but I'm not sure I follow the intent here, since "left" can't apply to the cross axis with the default "row" `flex-direction` and `justify-contents` is set above for content alignment.